iphone/Maps/Common/Common.swift

```diff
@@ -27,18 +28,42 @@ func statusBarHeight() -> CGFloat {
   return min(statusBarSize.height, statusBarSize.width)
 }
 
+private let enableLoggingInRelease = true
+
 func LOG(_ level: LogLevel,
          _ message: @autoclosure () -> Any,
          functionName: StaticString = #function,
          fileName: StaticString = #file,
          lineNumber: UInt = #line) {
-  if (Logger.canLog(level)) {
-    let shorFileName = URL(string: "\(fileName)")?.lastPathComponent ?? ""
-    let formattedMessage = "\(shorFileName):\(lineNumber) \(functionName): \(message())"
+
+  let shortFileName = URL(string: "\(fileName)")?.lastPathComponent ?? ""
+  let formattedMessage = "\(shortFileName):\(lineNumber) \(functionName): \(message())"
+
+  if #available(iOS 14.0, *), enableLoggingInRelease {
+    os.Logger.logger.log(level: OSLogLevelFromLogLevel(level), "\(formattedMessage, privacy: .public)")
+  } else if Logger.canLog(level) {
     Logger.log(level, message: formattedMessage)
```

Comment:
Does it make sense to also log into this logger on iOS 14+?

Reply:
We are using this logger for swift to pass it to the core logger. But there only 2 places where the logging from swift is used now.
I don't think that it's useful to duplicate logging messages. But of them will be visible in the debugger.

Reply:
Can duplication be avoided? What would be the best way to switch and use only one logger that can collect logs into a file when necessary and manually share this log with us for debugging?